tests/check-strings/check-strings

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python3
+
+# This script checks for strings like paths or class names that are *not* in the source code, but e.g. in
+# translation files, stylesheets or git files.
+# Invalid strings *in the source code* are usually recognized when you compile them, but other strings may
+# be overseen, which is why this script checks strings *outside of the source code*.
+
+
+import os.path
+import re
+import subprocess
+import xml.etree.ElementTree as ElementTree
+from pathlib import Path
+import tinycss2
+
+
+# global variables
+
+errors = 0
+
+
+# functions
+
+def caption(my_str):
+    print('')
+    print('# ' + my_str)
+    print('')
+
+def error(where, my_str):
+    global errors
+    errors += 1
+    print('Error: ' + where + ': ' + my_str)
+
+
+# prepare some variables
+
+if not os.path.exists(".gitmodules"):
+    print('You need to call this script from the LMMS top directory')
+    exit(1)
+
+result = subprocess.run(['git', 'ls-files', 'src/', 'include/', 'plugins/'], capture_output=True, text=True, check=True)
+files = result.stdout.splitlines()
+if os.path.isdir('plugins/carlabase/carla'):
+    carlabase = 'carlabase'
+else:
+    carlabase = 'CarlaBase'
+result = subprocess.run(['git',
+                         '--git-dir',   'plugins/'+carlabase+'/carla/.git',
+                         '--work-tree', 'plugins/'+carlabase+'/carla/',
+                         'ls-files', 'resources/ui', 'source/frontend'], capture_output=True, text=True, check=True)
+files_carla = ['plugins/CarlaBase/carla/' + i for i in result.stdout.splitlines()]
```

Comment:
- Why don't you put `'plugins/'+carlabase+'/carla` in a variable?
- Why you used `'plugins/CarlaBase/carla/'` instead of ``'plugins/'+carlabase+'/carla`` when collecting paths?